### PR TITLE
Wish to update the kwave_adapter file

### PR DESCRIPTION
**Pull request by JSackey101**
Added 2D/Line element support 
- Case 1 is the original code which supports 3D and position elements
First time I've done this so tell me if I need to add the test code I used for the other 3 cases + if I need to seriously annotate each test code I provide
Works by adding 1 more variable to the kwave_adapter input at the end - 1 to 4 to represent each test case described in the code
Signed-off-by: Jeffrey Sackey zcemjna@ucl.ac.uk

### CODE REVIEW

File: pacfish_matlab/@kwave_adapter/kwave_adapter.m [hunk 1]
```diff
@@ -1,101 +1,390 @@
...
+                end
+                
+                
+                
+                %% FIXME: we need a FOV definition.
+                if any(obj.fov)
+                    device_struct.general.field_of_view = obj.fov;
+                else
+                    device_struct.general.field_of_view = [];
+                end
```
Comment (jgroehl) [inline pacfish_matlab/@kwave_adapter/kwave_adapter.m:356]: Should we maybe consider creating a standardised default definition?
E.g. in #34

**Review by jgroehl — APPROVED** [review]
Looking good and is well documented!

**Comment by jgroehl** [comment]
Thanks!

File: pacfish_matlab/@kwave_adapter/kwave_adapter.m (hunk 1 above)
Comment (JSackey101) [reply pacfish_matlab/@kwave_adapter/kwave_adapter.m:356]: Just checking, by that you mean we would determine what would be used as the field of view if none is given (aka when else is triggered) ?

**Pull request merged**